kano_updater/scenarios.py

```diff
@@ -494,6 +494,12 @@ def enable_audio_device():
                 logger.error("end_config_transaction not present - update to kano-settings failed?")
         enable_audio_device()
 
+        # tell kano-overworld to skip onboarding stage
+        run_cmd_log('/usr/bin/luajit /usr/share/kano-overworld/bin/skip-onboarding.lua', unsudo=True)
+
+        # tell dashboard to skip Overworld and kit setup onboarding phase
+        run_cmd_log('touch ~/.dashboard-onboarding-vid-only', unsudo=True)
```

Comment:
This will need to use [`run_for_every_user()`](https://github.com/KanoComputing/kano-updater/blob/17f595026e5dfd6e35bf86bcf92d2714a6a3bc11/kano_updater/utils.py#L84-L86) instead as this will only touch the file for the user performing the update, not everyone else.

Also, can we change the name of the file to something along the lines of `~/.dashboard-click-onboarding-done` so that it makes sense for a new user.